spec/ADAS/ADAS.vspec

```diff
@@ -200,6 +200,45 @@ ESC.IsEngaged:
   type: sensor
   description: Indicates if ESC is currently regulating vehicle stability. True = Engaged. False = Not Engaged.
 
+ESC.IsCrossWindDetected:
+  datatype: boolean
+  type: sensor
+  description: Indicates if ESC system is detecting strong cross winds.
+               True = Strong Cross winds detected.
+               
+ESC.RoadFriction:
+  type: branch
+  description: Road friction values reported by ESC system.
+
+ESC.RoadFriction.Median:
```

Comment:
Is that fitting? The median doesn't have to be the most probable value (https://en.wikipedia.org/wiki/Normal_distribution). Would it be more the mode then?

Reply:
But "mode" is only for discrete numbers, right? But here we have continuous/float numbers.

I actually do not know what is needed for the use case. Assume we have the 5 calculations below:

`[1, 2, 7, 45, 45]`

Do we then want median (i.e. 7), or mean (i.e. 20), or the most probable value (45)

Reply:
I would probably go with the term "most probable", leaving the exact interpretation and derivation to the implementation of the road friction estimation.